frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,36 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
+                            <input id="old_password" name="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" required focus-if>
                             <span class="form-icon form-icon-dark"><i class="fa fa-user"></i></span>
                             <label for="old_password" data-error="wrong" data-success="right">Old Password*</label>
-                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password">{{changepwd.wrnMsg.old_password}}</div>
+                            <div class="wrn-msg text-highlight" ng-show="changepwd.isValid.old_password" ng-messages="changepwdForm.old_password.$error">{{changepwd.wrnMsg.old_password}}
```

Comment:
I have forgot to improve this in previous review.
remove `ng-show` attribute and text `{{changepwd.wrnMsg.old_password}}` and update code like this. 
`ng-if="changepwdForm.old_password.$touched || changepwdForm.$submitted" ng-messages="changepwdForm.old_password.$error"`
ref: `signup/ Reset Password Confirm'